src/common/FloatingButton.tsx

```
@@ -18,11 +18,25 @@ const FloatingButton = () => {
     }, 2500);
   };
 
+  const handleFloatingButtonClick = () => {
+    // Amplitude 이벤트 - 플로팅 버튼 클릭
+    if (typeof window !== 'undefined' && 'amplitude' in window && window.amplitude) {
+      (window as any).amplitude.track('todo_create', {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)